### DISCUSSION TITLE
What's the best way to ask for a fix or feature to be backported to a previous version or make sure it gets included in the next feature release?

### DISCUSSION THREAD

**Discussion by zakkak** [oracle/graal#4332] [Q&A] [answered]
Hello,

There are cases where we want a PR (often a fix) to get backported to the next minor release of an LTS version.
There are also cases where we would like to see a PR landing in the next release and thus we would like it to get reviewed and merged before the [code freeze date](https://github.com/oracle/graal/discussions/3509#discussioncomment-975794).

What's the best way to let the GraalVM team know about such requests?

**Comment by ezzarghili** [comment, marked as answer]
Hi

I had a discussion with @gilles-duboscq and others about this.

The current approach would be to create a ticket with details of the PR you want a backport for, mention @gilles-duboscq and @ezzarghili in your issue to bring attention to it if release code freeze date is near (~within 15 days of the deadline), we can still take in backports exceptionally after if it is critical enough and release date will not be compromised.

Backports will require the component lead approval.

The more critical a bug fix is the better chance it will get approved, the small bug fixes that have low chance of breaking something can also be approved as long as they are filed long before the code freeze date.

Once the ticket is approved for a backport, if you are the author of the original fix you can proceed with creating a PR targeting the public release branch, a sustaining engineer will take care of the integration internally and come back in case of any questions or issues, otherwise it would be good to get the original author take care of the PR or at least review it, approved PRs from Oracle will be handled the same as other internal backports.

We will create a special issue template for these requests so it can be used for these requests.
